build.ps1

```diff
@@ -41,10 +41,10 @@ Copy-Item ".\CreateDLL\bin\Release\NugetForUnity.dll" ".\Packager\Assets\NuGet\E
 Copy-Item ".\CreateDLL\bin\Release\DotNetZip.dll" ".\Packager\Assets\NuGet\Editor"
 
 # Launch Unity to export the NuGetForUnity package
-Start-UnityEditor -Project ".\Packager" -BatchMode -Quit -Wait -ExportPackage "Assets/NuGet .\NuGetForUnity.unitypackage" -LogFile ".\Packager\NuGetForUnity.unitypackage.log"
+Start-UnityEditor -Project ".\Packager" -BatchMode -Quit -Wait -ExecuteMethod "NugetForUnity.Export.Execute" -LogFile ".\Packager\NuGetForUnity.unitypackage.log"
```

Comment:
For an unknown reason, exporting packages via -ExportPackage do not work for me. To ensure this doesn't happen to others I've moved to using a script method which does the same thing.